catalog/camel-catalog/src/generated/resources/org/apache/camel/catalog/schemas/camel-spring.xsd

```diff
@@ -3973,50 +3973,10 @@ Enables random backoff. Default value: false
     <xs:complexContent>
       <xs:extension base="tns:noOutputDefinition">
         <xs:sequence/>
-        <xs:attribute name="ref" type="xs:string">
-          <xs:annotation>
-            <xs:documentation xml:lang="en">
-<![CDATA[
-Sets a reference to an existing bean to use, which is looked up from the registry.
-]]>
-            </xs:documentation>
-          </xs:annotation>
-        </xs:attribute>
-        <xs:attribute name="method" type="xs:string">
-          <xs:annotation>
-            <xs:documentation xml:lang="en">
-<![CDATA[
-Sets the method name on the bean to use.
-]]>
-            </xs:documentation>
-          </xs:annotation>
-        </xs:attribute>
-        <xs:attribute name="beanType" type="xs:string">
-          <xs:annotation>
-            <xs:documentation xml:lang="en">
-<![CDATA[
-Sets the class name (fully qualified) of the bean to use.
-]]>
-            </xs:documentation>
-          </xs:annotation>
-        </xs:attribute>
-        <xs:attribute name="scope" type="xs:string">
-          <xs:annotation>
-            <xs:documentation xml:lang="en">
-<![CDATA[
-Scope of bean. When using singleton scope (default) the bean is created or looked up only once and reused for the
-lifetime of the endpoint. The bean should be thread-safe in case concurrent threads is calling the bean at the same
-time. When using request scope the bean is created or looked up once per request (exchange). This can be used if you
-want to store state on a bean while processing a request and you want to call the same bean instance multiple times
-while processing the request. The bean does not have to be thread-safe as the instance is only called from the same
-request. When using prototype scope, then the bean will be looked up or created per call. However in case of lookup then
-this is delegated to the bean registry such as Spring or CDI (if in use), which depends on their configuration can act
-as either singleton or prototype scope. So when using prototype scope then this depends on the bean registry
-implementation. Default value: Singleton
-]]>
-            </xs:documentation>
-          </xs:annotation>
-        </xs:attribute>
+        <xs:attribute name="ref" type="xs:string"/>
+        <xs:attribute name="method" type="xs:string"/>
+        <xs:attribute name="beanType" type="xs:string"/>
+        <xs:attribute name="scope" type="xs:string"/>
```

Comment:
All of this needs to be removed from the PR. Please rebase the code and make sure this is not included.